lib/program/session.js

```diff
@@ -129,6 +129,11 @@ export default class Session {
       const operation = this.operation[key];
       const isLastOp = i === (sequence.length - 1);
 
+      utils.assert(
+        !!operation,
+        'Session: Unable to run operation. You need to initiate a session before this.',
```

Comment:
Let's add name of operation in message eg `Session: Unable to run uninitialized operation ExampleOperation:1.`